docs/en/learning_content/toolkit_basics_guides.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: default
+title: Toolkit Basics Guides
+nav_order: 1
+has_children: true
+parent: Learning Content
+permalink: /learning_content/toolkit_basics_guides
```

Comment:
@tannazie heads up that your pemalinks need to end with a `/` otherwise things will not work optimally!